app/main/views/services.py

```diff
@@ -248,3 +248,38 @@ def get_archived_service(archived_service_id):
     ).first_or_404()
 
     return jsonify(services=service.serialize())
+
+
+@main.route(
+    '/services/<string:service_id>/status/<string:status>',
```

Comment:
I think leave it explicitly declared; it's philosophically self-evident what variable we're expecting and it's the way we've declared everything else.